### ISSUE TITLE
Flaky TestHiveIntegrationSmokeTest.testVirtualBucketing

### ISSUE THREAD

**Issue by rschlussel** [prestodb/presto#13152]
Build failed with
```
[ERROR] testVirtualBucketing(com.facebook.presto.hive.TestHiveIntegrationSmokeTest)  Time elapsed: 0.927 s  <<< FAILURE!
java.lang.AssertionError: 
For query: 
 SELECT COUNT(DISTINCT("$path")) FROM test_virtual_bucket
not equal
Actual rows (up to 100 of 1 extra rows shown, 1 rows in total):
    [2]
Expected rows (up to 100 of 1 missing rows shown, 1 rows in total):
    [4]
	at org.testng.Assert.fail(Assert.java:94)
	at com.facebook.presto.tests.QueryAssertions.assertEqualsIgnoreOrder(QueryAssertions.java:223)
	at com.facebook.presto.tests.QueryAssertions.assertQuery(QueryAssertions.java:203)
	at com.facebook.presto.tests.QueryAssertions.assertQuery(QueryAssertions.java:119)
	at com.facebook.presto.tests.AbstractTestQueryFramework.assertQuery(AbstractTestQueryFramework.java:155)
	at com.facebook.presto.hive.TestHiveIntegrationSmokeTest.testVirtualBucketing(TestHiveIntegrationSmokeTest.java:2903)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by rschlussel** [comment]
@jessesleeping Do you know where these values are coming from?

**Comment by shixuan-fan** [comment]
@rschlussel I'll discuss this with @jessesleeping once he's back (Thursday hopefully).

**Comment by rschlussel** [comment]
Fixed in #13172
